src/opendr/perception/object_detection_2d/nanodet/README.md

```diff
@@ -10,4 +10,4 @@ Large parts of the implementation are taken from [Nanodet Github](https://github
 Usage
 ------
 - For VOC and COCO like datasets, an ```ExternalDataset``` with the root path and dataset name (```voc```, ```coco```) must be passed to the fit function.
-- The ```temp_path``` folder is used to save checkpoints during training.
\ No newline at end of file
+- The ```workspace``` folder is used to save checkpoints during training.
```

Comment:
What does `workspace` refer to? Is it something different to the `temp_path` that is used as an argument in the learner and is documented respectively.

Reply:
The default value of paremeter `temp_path` is `workspace`. This can be seen in config file of each model.
Is it preferable to change the default value to temp_path so it easier to understand?

Reply:
Default value in other learners is `temp` so it would be good to make it consistent with the others.